### DISCUSSION TITLE
Cannot get the app to open on Apple Silicon macOS Monterey

### DISCUSSION THREAD

**Discussion by CanfieldMobileDev** [pawelsalawa/sqlitestudio#4424] [General]
### Details
The app does not open after trying multiple times to right click + Open. The error says:

"The application “SQLiteStudio” can’t be opened."

### Steps to reproduce
Trying to open the app

### Operating system
macOS Monterey Version 12.2.1 (21D62)

### SQLiteStudio version
v3.3.3

**Comment by pawelsalawa** [comment]
In the dialog saying "The application “SQLiteStudio” can’t be opened." there should be an additional button, I think somewhere at the bottom, to open the application regardless of the warning.

**Comment by CanfieldMobileDev** [comment]
I tried to open multiple times, but this is all I get:


![image](https://user-images.githubusercontent.com/61331643/160444113-0d5290b9-dde3-4d6d-985c-8f863f134859.png)


Please note I am trying this on Apple Silicon M1 machine, not Intel. Is it even supported on a non-Intel platform?

**Comment by pawelsalawa** [comment]
In that case you oumay need to run it with the ix86 emulation. I don't know how that works exactly, but I know that it's possible on M1 macs.

I'm not Mac expert and I cannot help you more than that. Sorry.

**Comment by marcardar** [comment]
It works fine on my new MBP (M1) 12.1 - I only had to do the right-click + Open thing.

**Comment by jcpayne** [comment]
I noticed that someone has created an [M1 build](https://github.com/irmo-de/sqlitestudio-osx-arm64).

**Reply by pawelsalawa** [reply]
I think he or she builds it on his own M1 Mac. Support on GitHub actions is still not available, but - as far as I know - planned.
